tests/test_client_check.py

```diff
@@ -1,8 +1,121 @@
 import os
+import sys
 import pytest
-import subprocess
+import argparse
 import tempfile
+import subprocess
 import configparser
+from urlchecker.client.check import main as check_main
+from urlchecker.client.__init__ import main as init_main
```

Comment:
Usually you don't need to import from init, you just import from client (it's the same thing).
```suggestion
from urlchecker.client import main as init_main
```